src/main/kotlin/com/terraformation/backend/customer/db/OrganizationStore.kt

```diff
@@ -535,6 +535,8 @@ class OrganizationStore(
         .from(ORGANIZATION_USERS)
         .where(ORGANIZATION_USERS.ORGANIZATION_ID.eq(organizationId))
         .and(ORGANIZATION_USERS.ROLE_ID.eq(Role.TerraformationContact))
+        .orderBy(ORGANIZATION_USERS.USER_ID)
+        .limit(1)
```

Comment:
A future PR will add a `fetchTerraformationContacts` method and deprecate this one, so this is just a temporary change to keep backwards compatibility.